extensions/obviousAlexC/SensingPlus.js

```diff
@@ -614,10 +634,39 @@
           toggleMenu: {
             items: ["on", "off"],
           },
+          effectMenu: {

```

Comment:
all of these need acceptReporters: true,

Reply:
ok